### DISCUSSION TITLE
Updating SearXNG

### DISCUSSION THREAD

**Discussion by MrNobody334** [searxng/searxng#1499] [Q&A]
Is it necessary to update SearXNG? How to update it? It didn't work the update command in the SearXNG documentation. 
And how to update docker containers?

**Comment by mrpaulblack** [comment]
Hi @MrNobody334 ,
for the docker containers... You cannot really upgrade a running container AFAIK (or at least it would IMO not make sense...). What you can do is create a new container with a more recent version that replaces the old one. There ore build in tools for that. You can just run `docker-compose pull` and then `docker-compose up -d`.
It will first pull the latest version of the docker images defined in `docker-compose.yml` and then replace all running containers with newer ones, if there is a newer image version for that container.
